ReadMe.md

```diff
@@ -13,7 +13,7 @@ You can see all the packages and differences in the table below:
 | Package name                                | Latest Version | Content                                                                                                                | Usage Instructions                                                          |
 |---------------------------------------------|----------------|------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
 | LibMtpSharp                                 | 0.1.3          | Contains the wrapper managed code. API coverage is limited and will improve in future versions.                        | Main package with logic. Required to be able to use libmpt                  |
-| LibMtpSharpStandardMacOS                    | 0.1.6          | Contains the wrapper managed code with MacOS dependencies for .Net standard                                            | Main package with logic for .Net standard when you target MacOS             |
+| LibMtpSharpStandardMacOS                    | 0.1.8          | Contains the wrapper managed code with MacOS dependencies for .Net standard                                            | Main package with logic for .Net standard when you target MacOS             |
 | LibMtpSharp.Native.Linux                    | 1.1.20-beta    | Linux native Libmtp library with custom improvements built for x64                                                     | Use if you target linux and instruct the user how to install dependencies   |
 | LibMtpSharp.Native.Linux.WithDependencies   | 1.1.20-beta    | Linux native dependencies built for x64 and references LibMtpSharp.Native.Linux package.                               | Use if you target linux and don't want user to manage dependencies          |
 | LibMtpSharp.Native.MacOS                    | 1.1.20-beta    | MacOS native Libmtp library with custom improvements built for x64 (not sure if will work on M1)                       | Use if you target MacOS and instruct the user how to install dependencies   |
```

Comment:
I don't know if this somehow affects us, but we should test your library also on M1. If it won't be working, that it might be huge blocker for us.

Reply:
Dominik was testing detection of mtp devices on Mac with M1